static/app/views/settings/organizationMembers/inviteBanner.tsx

```diff
@@ -68,6 +70,20 @@ export function InviteBanner({
     });
   }, [api, organization, promptsFeature, hideBanner]);
 
+  useEffect(() => {
+    const params = new URLSearchParams(location.search);
+    const showModal = params.get('showModal');
+
+    if (!hideBanner && showModal) {
+      openInviteMissingMembersModal({
+        allowedRoles,
+        missingMembers,
+        organization,
+        onClose: onModalClose,
+      });
+    }
+  }, [allowedRoles, missingMembers, organization, onModalClose, location, hideBanner]);
```

Comment:
That's a lot of stuff that might re-open the modal if it changes reference.

Reply:
@cathteng is this actually expected? You're using this to pass params to openInviteMissingMembersModal but useEffect params are basically the ones you want to trigger useEffect on change, so only hideBanner. You can instead call a local with no params.

Reply:
call a local function with no params? wouldn't i still have to use `useCallback` with dependencies on these same variables?